splash/qtrender_lua.py

```diff
@@ -221,6 +248,10 @@ def init_dispatcher(self, return_func):
         """
         self._return = return_func
 
+    @attr_function(attr_name='js_enabled', default=True)
```

Comment:
Hi @sunu,

I think having default values this way is not good; it is better for property to be able to read an actual property value. This would be less error prone. Currently if we e.g. add a command-line option to use js_enabled=False by default, default value here will have to be changed, but it is not clear how. Also, other commands may change these properties; in this case the read value will be incorrect.